MediaGalleryRenditions/etc/media_content.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_MediaContentApi:etc/media_content.xsd">
+    <search>
+        <patterns>
+            <pattern name="renditions">/"?.renditions(.*?)"?}}/</pattern>
```

Comment:
Please ensure the pattern is covering all the way the image can be stored in the content.

Here is the current patterns from other modules:

```
            <pattern name="catalog_image">/^\/?media\/(.*)/</pattern>
            <pattern name="catalog_image_with_pub">/^\/pub\/?media\/(.*)/</pattern>
            <pattern name="media_gallery">/{{media url="?(.*?)"?}}/</pattern>
            <pattern name="wysiwyg">/src=".*\/media\/(.*?)"/</pattern>
```